src/enumerable.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield(self[i])
```

Comment:
- [ ] my_each returns an Enumerator if no block is given
- [ ] my_each_with_index returns an enumerator if no block is given